### ISSUE TITLE
not supported binding with name (DuckDB::Error)

### ISSUE THREAD

**Issue by pere73** [suketa/ruby-duckdb#612]
With `duckdb 0.10.1.0` the following code raises an error:

```
require 'duckdb'
db  = DuckDB::Database.open
con = db.connect
con.query('CREATE TABLE users (id INTEGER, name VARCHAR(30))')

[[1, 'Grace'], [2, 'Hank']].each do |id, name|
  con.query("INSERT into users VALUES ($id, $name)", id: id, name: name)
end
```

**Comment by suketa** [comment]
@pere73 
Thank you for your reporting.
I fixed it (#613) and I'll release fixed version later.

**Comment by suketa** [comment]
@pere73 
I released v0.10.1.1. Try it.

**Comment by pere73** [comment]
Thank you.. It's solved.
